src/plugins/ResizeReorder/ReorderHandle.js

```diff
@@ -0,0 +1,261 @@
+      </div>
+    );
+  }
+
+  onTouchStart = (ev) => {
+    if (!this.props.touchEnabled) {
+      return;
+    }
+    this.onMouseDown(ev);
+  };
+
+  onTouchEnd = (ev) => {
+    if (this.props.touchEnabled)
+      ev.stopPropagation();
+  };
+
+  onTouchMove = (ev) => {
+    if (this.props.touchEnabled)
+      ev.stopPropagation();
+  };
+
+  /**
+   *
+   * @param {MouseEvent} event
+   */
+  onMouseDown = (event) => {
+    this.cursorDeltaX = 0;
+    this.isReordering = true;
+    this.scrollStart = getState().scrollX;
+    this.originalLeft = this.props.left;
+    this.initializeDOMMouseMoveTracker(event);
+    this.updateParentReorderingData({
+      isColumnReordering: true,
+      displacement: 0
+    });
+    this.frameId = requestAnimationFrame(this.updateDisplacementPeriodically);
+  };
+
+  /**
+   * @param {number} deltaX
+   */
+  onMouseMove = (deltaX) => {
+    this.cursorDeltaX += deltaX * (this.props.isRTL ? -1 : 1);
+  };
+
+  onMouseUp = () => {
+    cancelAnimationFrame(this.frameId);
+    this.updateColumnOrder();
+    this.isReordering = false;
+    this.frameId = null;
+    this.cursorDeltaX = 0;
+    this.mouseMoveTracker.releaseMouseMoves();
+    this.updateParentReorderingData({
+      isColumnReordering: false,
+      displacement: 0
+    });
+  };
+
+  /**
+   *
+   * @param {MouseEvent} event
+   */
+  initializeDOMMouseMoveTracker = (event) => {
+    this.mouseMoveTracker = new DOMMouseMoveTracker(
+      this.onMouseMove,
+      this.onMouseUp,
+      document.body,
+      this.props.touchEnabled
+    );
+    this.mouseMoveTracker.captureMouseMoves(event);
+  };
+
+  updateParentReorderingData = (state) => {
+    this.props.updateParentReorderingData(state);
+  };
+
+  updateDisplacementPeriodically = () => {
+    /* NOTE: We need to use requestAnimationFrame because whenever column reaches the end of table (scroll width is left),
+     we want to update the scrollX which can't be updated if we uer onMouseMove*/
+    this.frameId = requestAnimationFrame(this.updateDisplacementPeriodically);
+    this.updateDisplacementWithScroll();
+  };
+
+  updateDisplacementWithScroll = () => {
+    const scrollStart = this.scrollStart;
+    let { isFixed } = this.props;
+    let { scrollX, maxScrollX } = getState();
+    let deltaX = this.cursorDeltaX;
+    if (!isFixed) {
+      // Relative dragX position on scroll
+      const dragX = this.originalLeft - scrollStart + deltaX;
+      const availableScrollWidth = this.props.availableScrollWidth;
+      deltaX += scrollX - scrollStart;
+
+      // Scroll the table left or right if we drag near the edges of the table
+      if (dragX > availableScrollWidth - DRAG_SCROLL_BUFFER) {
+        scrollX = Math.min(scrollX + DRAG_SCROLL_SPEED, maxScrollX);
+      } else if (dragX <= DRAG_SCROLL_BUFFER) {
+        scrollX = Math.max(scrollX - DRAG_SCROLL_SPEED, 0);
+      }
+      this.props.scrollToX(scrollX);
+    }
+    this.updateParentReorderingData({ displacement: deltaX });
```

Comment:
Update displacement wrt scroll position. It also:
1. Informs ResizeReorderCell to translate the div. (line 194)
2. Dispatches action to scroll to a specific position. (line 192)